app/src/main/kotlin/com/mapzen/erasermap/view/MainActivity.kt

```diff
@@ -408,21 +429,23 @@ public class MainActivity : AppCompatActivity(), MainViewController, RouteCallba
             Tangram.addDataSource(searchResults);
         }
 
-        var activeFeature: Int = 0
+        var featureCount: Int = 0
         searchResults?.clear()
         for (feature in features) {
             val simpleFeature = SimpleFeature.fromFeature(feature)
             val lngLat = LngLat(simpleFeature.lng(), simpleFeature.lat())
             val properties = com.mapzen.tangram.Properties()
+            properties.add("tap", "search")
```

Comment:
Great thanks!

See https://github.com/mapzen/eraser-map/blob/master/app/src/main/kotlin/com/mapzen/erasermap/view/InstructionListActivity.kt#L12 for an example of how to declare constants using a `companion object` and the `@JvmStatic` annotation.